tests/functions/test_base.py

```diff
@@ -1,10 +1,33 @@
+import os
+
+import mock
 import pytest
 
 from exodus_lambda.functions.base import LambdaBase
 
-CONF_PATH = "exodus_lambda/functions/lambda_config.json"
+CONF_PATH = "configuration/lambda_config.json"
 
 
-def test_base():
+def test_base_handler():
     with pytest.raises(NotImplementedError):
         LambdaBase(conf_file=CONF_PATH).handler(event=None, context=None)
+
+
+@pytest.mark.parametrize(
+    "env_var,exp_var",
+    [
+        ("us-south-7", "us-east-1"),
+        ("ap-southeast-2", "ap-southeast-2"),
+        ("ap-northeast-3", "us-east-1"),
+    ],
+    ids=["fake", "available", "unavailable"],
+)
+def test_base_region(env_var, exp_var):
+    """Ensure correct regions are selected for various inputs"""
+    with mock.patch.dict(os.environ, {"AWS_REGION": env_var}):
```

Comment:
Doesn't really matter here but I want to point out that there is a `monkeypatch` fixture which in many cases is a better way to temporarily override env vars.  https://docs.pytest.org/en/latest/monkeypatch.html

```monkeypatch.setenv("AWS_REGION", env_var)```

The reason why it would usually be better is that it keeps the rest of the environment as it is, while here you're wiping out all the rest of the environment, so AWS_REGION is the only present variable which is quite unrealistic.  But I don't think it will have an impact on this particular test.

Reply:
Fixed